cclib/method/ddec.py

```diff
@@ -430,162 +504,160 @@ def condition_densities(self):
-                        elif 3 * abs(belowphi) < abs(abovephi):
-                            corbigPhi = roots.min() - 2.0 * belowphi * (
-                                roots.max() - roots.min()
-                            ) / (abovephi - belowphi)
-                        else:
-                            corbigPhi = (roots.max() + roots.min()) / 2.0
-                        # New candidates
-                        corphi = self._phiai(self._y_a[atomi], corbigPhi, atomi)[0]
-                        self._candidates_bigPhi[atomi] = numpy.array(
-                            [
-                                lowerbigPhi,
-                                midbigPhi,
-                                upperbigPhi,
-                                roots.max(),
-                                roots.min(),
-                                corbigPhi,
-                            ],
-                            dtype=float,
-                        )
-                        self._candidates_phi[atomi] = numpy.array(
-                            [lowerphi, midphi, upperphi, abovephi, belowphi, corphi], dtype=float
-                        )
-
-                        # Update upperphi and lowerphi
-                        lower_ind = numpy.where(
-                            self._candidates_phi[atomi]
-                            == self._candidates_phi[atomi][self._candidates_phi[atomi] < 0].max()
-                        )[0][0]
-                        upper_ind = numpy.where(
-                            self._candidates_phi[atomi]
-                            == self._candidates_phi[atomi][self._candidates_phi[atomi] > 0].min()
-                        )[0][0]
-
+                        # Fitting needs to continue in this case.
+                        lowerbigPhi = self._candidates_bigPhi[atomi][lower_ind]
                         lowerphi = self._candidates_phi[atomi][lower_ind]
+                        upperbigPhi = self._candidates_bigPhi[atomi][upper_ind]
                         upperphi = self._candidates_phi[atomi][upper_ind]
 
-                        if abs(lowerphi) < 1e-10:
-                            bigphiAI[atomi] = self._candidates_bigPhi[atomi][lower_ind]
-                            fitphi = False
-                        elif abs(upperphi) < 1e-10:
-                            bigphiAI[atomi] = self._candidates_bigPhi[atomi][upper_ind]
-                            fitphi = False
-                        else:
-                            # Fitting needs to continue in this case.
-                            lowerbigPhi = self._candidates_bigPhi[atomi][lower_ind]
-                            lowerphi = self._candidates_phi[atomi][lower_ind]
-                            upperbigPhi = self._candidates_bigPhi[atomi][upper_ind]
-                            upperphi = self._candidates_phi[atomi][upper_ind]
-
-            assert not fitphi, "Iterative conditioning failed to converge."
+                if iteration == 49:
```

Comment:
Can this be an argument with a  default value of 49

Reply:
I've just modified this. Thanks!